packages/emails/email-manager.ts

```diff
@@ -88,7 +88,8 @@ export const sendRescheduledEmails = async (calEvent: CalendarEvent) => {
 
   emailsToSend.push(
     ...calEvent.attendees.map((attendee) => {
-      return sendEmail(() => new AttendeeRescheduledEmail(calEvent, attendee));
+      const clonedCalEvent = cloneDeep(calEvent);
+      return sendEmail(() => new AttendeeRescheduledEmail(clonedCalEvent, attendee));
```

Comment:
Most attendee emails are extended from `AttendeeScheduledEmail` which has logic to filter out other attendee information if the event is a seats event & show attendees is false. Since it makes edits to the event we have to create a new cloned event for every attendee.

What was happening before was for the first attendee, the event attendees would be filtered. When moving on the to second attendee it would receive the same event as the first attendee with the filtered attendees' array.